news-bundle/config/services.yaml

```diff
@@ -30,9 +48,22 @@ services:
         class: Contao\NewsBundle\EventListener\InsertTagsListener
         arguments:
             - '@contao.framework'
+            - '@logger'
         tags:
             - { name: contao.hook, hook: replaceInsertTags }
 
+    contao_news.listener.news_feed:
+        class: Contao\NewsBundle\EventListener\NewsFeedListener
+        arguments:
+            - '@contao.framework'
+            - '@contao.image.factory'
+            - '@contao.insert_tag.parser'
+            - '%kernel.project_dir%'
+            - '@contao.cache.entity_tags'
+        tags:
+            - { name: kernel.event_listener, event: Contao\NewsBundle\Event\FetchArticlesForFeedEvent, method: onFetchArticlesForFeed }
+            - { name: kernel.event_listener, event: Contao\NewsBundle\Event\TransformArticleForFeedEvent, method: onTransformArticleForFeed }
```

Comment:
Please use PHP 8 attributes to tag the listener.

Reply:
Fixed in d8ddd2965